models/banner.js

```diff
@@ -0,0 +1,130 @@
+'use strict';
+
+const Joi = require('joi');
+const mutate = require('../lib/mutate');
+
+const MODEL = {
+    id: Joi
+        .number()
+        .integer()
+        .positive(),
+
+    message: Joi
+        .string()
+        .max(512)
+        .description('Body of banner to display')
+        .example('Due to planned upgrade of Kubernetes, Screwdriver will be down'),
+
+    isActive: Joi
+        .boolean()
+        .description('Flag if the banner is active')
+        .example(true)
+        .default(false),
+
+    dateCreated: Joi
+        .string()
+        .isoDate()
+        .description('When this banner was created')
+        .example('2017-01-06T01:49:50.384359267Z'),
+
+    createdBy: Joi
+        .number()
+        .integer()
+        .positive()
+        .description('ID of user creating the banner'),
+
+    type: Joi
+        .string().valid([
+            'info',
+            'warn'
+        ])
+        .description('Type/Severity of the banner message')
+        .example('info')
+};
+
+module.exports = {
+    /**
+     * All the available properties of Banners
+     *
+     * @property base
+     * @type {Joi}
+     */
+    base: Joi.object(MODEL).label('Banner'),
+
+    /**
+     * Properties for Banner that will come back during a GET request
+     *
+     * @property get
+     * @type {Joi}
+     */
+    get: Joi.object(mutate(MODEL, [
+        'id'
```

Comment:
If these are fields that are returned during a GET request, shouldn't it at least also include everything passed in for the CREATE (`message`, `type`)?